indico/modules/events/evaluation/tasks.py

```diff
@@ -0,0 +1,76 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from celery.schedules import crontab
+
+from indico.core.celery import celery
+from indico.core.db import db
+from indico.core.notifications import make_email, send_email
+from indico.modules.events.evaluation import event_settings as evaluation_settings
+from indico.util.date_time import now_utc
+from indico.web.flask.templating import get_template_module
+from MaKaC.evaluation import Evaluation
+
+
+@celery.periodic_task(name='evaluation_start_notifications', run_every=crontab(minute='0', hour='0'))
+def notify_evaluation_start():
+    for setting in evaluation_settings.query.filter_by(name='send_notification'):
+        if not setting.value:
```

Comment:
On Postgres 9.4 (maybe 9.3, too, not sure) this could actually go into the query using `value::jsonb @> 'true'::jsonb`

Too bad we are currently stuck with 9.2...